.appveyor.yml

```diff
@@ -22,7 +22,7 @@ install:
   - 7z x sfdx-windows-amd64.tar.xz
   - 7z x sfdx-windows-amd64.tar
   - SET PATH=%APPVEYOR_BUILD_FOLDER%\sfdx\bin;%PATH%
-  - sfdx update
+  - sfdx update alpha
```

Comment:
We will remove the alpha once R2 goes out since then it would be the default and no longer alpha. Ditto for the other CI files.